### ISSUE TITLE
java.lang.OutOfMemoryError: unable to create new native thread

### ISSUE THREAD

**Issue by youest** [sbtourist/Journal.IO#9]
Hi Sergio,
I've need to open many journal at the same time but I can't do it because the journal executes a thread for each.

As you can see running this simple test:

```

import journal.io.api.Journal;

/**
 */
public class OutOfMemoryTest {

    public static void main(String[] args) {
        int count = 0;
        while (true) {
            count++;
            Journal journal = new Journal();
            try {
                journal.open();

            } catch (Throwable e) {
                e.printStackTrace();
                break;

            }

        }
        System.out.println(count);
    }
}

```

This behavior clashes with the operative system limits.

Best regards,
Giuseppe.

**Comment by sbtourist** [comment]
Thanks, I'll look into this ASAP and come back with more info.

**Comment by sbtourist** [comment]
Closed, see issue #10.
